cg/meta/report/balsamic.py

```diff
@@ -80,34 +88,38 @@ def get_panel_metadata(
         return BalsamicTargetedSampleMetadataModel(
             bait_set=bed.name if bed else None,
             bait_set_version=analysis_metadata.config.panel.capture_kit_version,
-            million_read_pairs=million_read_pairs,
+            duplicates=sample_metrics.percent_duplication if sample_metrics else None,
+            fold_80=sample_metrics.fold_80_base_penalty if sample_metrics else None,
+            gc_dropout=sample_metrics.gc_dropout if sample_metrics else None,
+            mean_insert_size=sample_metrics.mean_insert_size if sample_metrics else None,
             median_target_coverage=(
                 sample_metrics.median_target_coverage if sample_metrics else None
             ),
+            million_read_pairs=million_read_pairs,
             pct_250x=sample_metrics.pct_target_bases_250x if sample_metrics else None,
             pct_500x=sample_metrics.pct_target_bases_500x if sample_metrics else None,
-            duplicates=sample_metrics.percent_duplication if sample_metrics else None,
-            mean_insert_size=sample_metrics.mean_insert_size if sample_metrics else None,
-            fold_80=sample_metrics.fold_80_base_penalty if sample_metrics else None,
-            gc_dropout=sample_metrics.gc_dropout if sample_metrics else None,
+            reception_qc_flag=reception_qc_flag,
         )
 
     @staticmethod
     def get_wgs_metadata(
-        million_read_pairs: float, sample_metrics: BalsamicWGSQCMetrics
+        million_read_pairs: float,
+        reception_qc_flag: ReceptionQCFLag,
+        sample_metrics: BalsamicWGSQCMetrics,
     ) -> BalsamicWGSSampleMetadataModel:
         """Return report metadata for Balsamic WGS analysis."""
         return BalsamicWGSSampleMetadataModel(
-            million_read_pairs=million_read_pairs,
+            duplicates=sample_metrics.percent_duplication if sample_metrics else None,
+            fold_80=sample_metrics.fold_80_base_penalty if sample_metrics else None,
+            mean_insert_size=sample_metrics.mean_insert_size if sample_metrics else None,
             median_coverage=sample_metrics.median_coverage if sample_metrics else None,
+            million_read_pairs=million_read_pairs,
             pct_15x=sample_metrics.pct_15x if sample_metrics else None,
             pct_60x=sample_metrics.pct_60x if sample_metrics else None,
-            duplicates=sample_metrics.percent_duplication if sample_metrics else None,
-            mean_insert_size=sample_metrics.mean_insert_size if sample_metrics else None,
-            fold_80=sample_metrics.fold_80_base_penalty if sample_metrics else None,
             pct_reads_improper_pairs=(
                 sample_metrics.pct_pf_reads_improper_pairs if sample_metrics else None
             ),
+            reception_qc_flag=reception_qc_flag,
```

Comment:
This can be none as well following the function above, perhaps it would be nice to have this explicitly stated:
```suggestion
            reception_qc_flag=reception_qc_flag if reception_qc_flag else None,
```

Reply:
I think it's redundant for the `reception_qc_flag`. The check is necessary for the other metrics since we are accessing some attributes from the sample metrics, which could be empty